src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -286,8 +286,11 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
 
     val proxyGroupDeletion = userF.flatMap(gcsDAO.deleteProxyGroup)
 
+    val tokenRemoval = toFutureTry(gcsDAO.revokeToken(userRef) flatMap { _ => gcsDAO.deleteToken(userRef) })
```

Comment:
Why `toFutureTry`?  That pattern is not used here.  We *could* use it, but in that case I'd prefer to use it for the rest of the method too, for consistency.